GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -310,7 +311,7 @@ private void ReloadCommitInfo()
 
             if (_revision is not null && !_revision.IsArtificial)
             {
-                StartAsyncDataLoad();
+                StartAsyncDataLoad(Module.EffectiveSettings);
```

Comment:
We need to take a copy of effective settings, otherwise, as tests show, a `Module` can get disassociated from a control (e.g. get disposed) before the async operation gets complete. By making a local copy we can complete the async operation without any issues.